cookiecutter.json

```diff
@@ -10,5 +10,6 @@
     "email_password": "",
     "sentry_dsn": "",
     "app_name": "things",
-    "model_name": "Thing"
+    "model_name": "Thing",
+    "model_name_lower": "{{ cookiecutter.model_name|lower }}"
```

Comment:
Yeah it will prompt. As far as I'm aware you cant add things to the context in hooks. If we drop it completely we are going to break windows (#7).